BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
+
+app.Run();
+
+public struct CnbRatesResponse
+{
+    [JsonPropertyName("rates")] public List<Rate> Rates { get; set; }
+}
+
+public struct Rate
+{
+    [JsonPropertyName("currencyCode")] public string CurrencyCode { get; set; }
+    [JsonPropertyName("rate")] public float RateValue { get; set; }
+}
```

Comment:
Create a folder which name “Models”. So we can always find easily what we are looking for. Then create 2 class inside “Models” folder. “Rates.cs”  and “CnbRatesResponse.cs”. We should prefer class instead of struct because  classes are better for larger, complex objects, while structs are best for small, simple data types. We can’t know what will be needed in the future. Keep it ready for extension.

Reply:
```
public class CnbRatesResponse
{
    [JsonPropertyName("rates")]
    public IReadOnlyList<Rate> Rates { get; set; }
}
```

Reply:
public class Rate
{
    [JsonPropertyName("currencyCode")]
    public string CurrencyCode { get; set; }

    [JsonPropertyName("rate")]
    public float RateValue { get; set; }
}

Reply:
It's good to use `JsonPropertyName`, to match variables from API.